snmp/datadog_checks/snmp/data/profiles/isilon.yaml

```diff
@@ -7,7 +10,62 @@ extends:
 device:
   vendor: "dell"
 
-sysobjectid: 1.3.6.1.4.1.12124.*
+sysobjectid:
+  - 1.3.6.1.4.1.12124.*
+
+  # Looks like EMC Isilon devices might use the wrong sysObjectID `1.3.6.1.4.1.12325.1.1.2.1.1` (FreeBSD-FOKUS)
+  # Few related discussions:
+  # - https://thwack.solarwinds.com/product-forums/network-performance-monitor-npm/f/forum/66098/emc-isilon-showing-up-as-non-emc-device-after-upgrade/39511
+  # - https://support.servicenow.com/kb?id=kb_article_view&sysparm_article=KB0761173
+  # - https://github.com/librenms/librenms/issues/6412
+  # Long term solution:
+  # Similar to librenms a possible solution is to have other ways to match devices like sysDescr text matching. See example:
+  # https://github.com/librenms/librenms/pull/6416/files#diff-ec0470a0c495f1349b349f62b0dfdf14b928a5acb94f0de5f6c5a6e523f7817cR12-R16
+  - 1.3.6.1.4.1.12325.1.1.2.1.1
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "dell"
+      serial_number:
+        symbol:
+          OID: 1.3.6.1.4.1.12124.2.51.1.3.1
```

Comment:
Could there be multiple chassis?

Reply:
From those two sources it seems there is only one chassis item:
1/ https://gist.github.com/dsgagi/1109e0c9d97044f7686947cf3ba9357c

2/ 
https://www.dell.com/community/Isilon/Querying-Isilon-Cluster-via-SNMP-For-Node-Model/m-p/7048624/highlight/true#M5298

```
++++++++++++++++++++++++++++++++++++

$ snmptable -v 2c -c XXXXXX -O0b -Cfil ----- -Ln -m ISILON-MIB X.X.X.X chassisTable | sed -E 's/,//g' | sed -E 's/-----/,/g' | sed -E 's/,+$//g'

SNMP table: ISILON-MIB::chassisTable

index,chassisNumber,chassisConfigNumber,chassisSerialNumber,chassisModel,chassisUnitIDLEDOn

1,1,400-0034-03,SX200-XXXXX,X200-2U-Single-12GB-2x1GE-2x10GE SFP+-12TB,na

++++++++++++++++++++++++++++++++++++
```